### PR TITLE
feat: [Python SDK]Raise Error for unsupported methods & new unittest for cicd

### LINKED ISSUE #888: Support standard Python SQLAlchemy API and throw exceptions for unsupported methods
Now we support some basic Python SQLAlchemy APIs but it is not documented. We should add documents for supported APIs and throw exception or print errors for unsupported methods.

### PR DESCRIPTION
**Pull request by HuilinWu2**
1, Raise NotSupportedError for all unsupported methods;
2, add new unittest for three examples APIs: Cursor.executemany(),Cursor.fetchmany(size=N),Cursor.fetchall().

### CODE REVIEW

**Comment by tobegit3hub** [comment]
Resolve https://github.com/4paradigm/OpenMLDB/issues/888 .

File: python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py [hunk 1]
```diff
@@ -304,7 +304,7 @@ def execute(self, operation, parameters=()):
 
     @connected
     def executemany(self, operation, parameters=()):
-        pass
+        raise NotSupportedError("executemany() is currently unsupported in OpenMLDB")
```
Comment (tobegit3hub) [inline python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py]: The exception message could be simpler. Just like ``raise NotSupportedError("Unsupported in OpenMLDB").

File: python/test/openmldb_client_test.py [hunk 2]
```diff
@@ -176,7 +181,28 @@ def check_result(self, rs, expectRows, orderIdx = 0):
...
       self.assertEqual(row, expectRows[i], "not equal row: {}\n{}".format(row, expectRows[i]))
       i+=1
-    
+
+  def check_exectute_many(self,connection,sql):
+    try:
+      connection.execute(sql,[{"col1":1005, "col2":"2020-12-29", "col3":"shandong", "col4":"jinan", "col5":6},
+                              {"col1":1006, "col2":"2020-12-30", "col3":"fujian", "col4":"fuzhou", "col5":7}]);
+    except Exception as e:
+      self.assertTrue(False)
```
Comment (tobegit3hub) [inline python/test/openmldb_client_test.py:189]: It seems that it always throws exception and trigger the code `self.assertTrue(False)` which will fail the tests.

Maybe we should change the code like this.

```
    try:
      connection.execute(sql,[{"col1":1005, "col2":"2020-12-29", "col3":"shandong", "col4":"jinan", "col5":6},
                              {"col1":1006, "col2":"2020-12-30", "col3":"fujian", "col4":"fuzhou", "col5":7}]);
      self.assertTrue(False)
    except Exception as e:
      pass
```

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

File: python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py (hunk 1 above)
Comment (HuilinWu2) [reply python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py]: Updated.

File: python/test/openmldb_client_test.py (hunk 2 above)
Comment (HuilinWu2) [reply python/test/openmldb_client_test.py:189]: Updated.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by dl239 — APPROVED** [review]

**Review by keyu813 — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[Make Python SDK available for users](https://trello.com/c/Mfqg2o9I/33-make-python-sdk-available-for-users)

**Pull request merged**